assets/css/edd-admin.css

```diff
@@ -2650,10 +2650,9 @@ body.download_page_edd-reports {
 }
 
 #edd-reports-tiles-wrap {
-	display: flex;
-	flex-wrap: wrap;
-	margin-left: 10px;
-	margin-right: 10px;
+	display: grid;
```

Comment:
I haven't used CSS grid, but is there a way to force the columns to be the same width? They are varying widths for me with this in place, depending on the tile content.

Reply:
I'm running into that, too.  I'm still playing with making them equal, but also not forcing there to be 3 columns on each row.

Reply:
Hey @robincornett.  I just did another push.  Let me know what you think about the sizing.

Reply:
@LisaCee gorgeous! It's working great for me on multiple screen sizes.

Note: with the change to CSS Grid, I think we can remove the `flex: 0 1 calc(33% - 20px);` from `.edd-reports-tile` since it's no longer using flexbox (the tile itself is still using it so everything else should stay, but I think that's the only one applying to the tile as a child of a flexbox container). Line 2664, it looks like.

Reply:
We can also now remove the `margin` in place of `grid-gap: 20px;` (and remove the `padding` override).

Reply:
New push to review.  `grid-gap: 20px` looked too big so I did`10px`, but let me know what you guys think.

Reply:
What about the `grid-gap: 20px`, and removing the margins set on `.edd-reports-tile`, which I think is part of the reason the tiles are causing issues and requiring the negative margins in the first place? It looks like the `grid-gap` property on the parent container makes the appearance of margins but without the extra math on the user end?

Reply:
@LisaCee Robin is correct we can remove the spacing on the individual tile. The `20px` previously was added on top of the `10px` left/right margin which would make it `40px` total. 

We also need to add `-ms-` prefixes for some of these rules. Some more info here: https://medium.com/@elad/supporting-css-grid-in-internet-explorer-b38669e75d66